packages/backend/src/clients/DockerHub/DockerHub.ts

```diff
@@ -0,0 +1,38 @@
+import fetch from 'node-fetch';
+
+let dockerHubVersion: string | undefined;
+
+const filterByName = (result: { name: string }): boolean =>
+    /[0-9.]+$/.test(result.name);
+
+const sorterByDate = (
+    a: { last_updated: string },
+    b: { last_updated: string },
+): number =>
+    Number(new Date(b.last_updated)) - Number(new Date(a.last_updated));
+
+async function fetchDockerHubVersion(): Promise<string | undefined> {
+    try {
+        const response = await fetch(
+            'https://hub.docker.com/v2/repositories/lightdash/lightdash/tags',
+            { method: 'GET' },
+        );
+        const version = (await response.json()).results
+            .filter(filterByName)
+            .sort(sorterByDate)[0].name;
+        return version;
+    } catch {
+        return undefined;
+    }
+}
+
+async function updateDockerHubVersion() {
+    const version = await fetchDockerHubVersion();
+    if (version) dockerHubVersion = version;
+}
+setInterval(updateDockerHubVersion, 10 * 60 * 1000); // 10 minutes
+updateDockerHubVersion();
```

Comment:
This needs an error handler or it'll crash the server!

```
updateDockerHubVersion()
  .catch(e => Logger.error('Some useful error for the logs');
```

Reply:
are we not cathing the `fetch` method already ? what error could happen on updateDockerHubVersion ?

Reply:
oh yes true! I didn't see that.

But we should make a habit of always handling exceptions on async functions. That way we don't rely on somebody to remember to catch every error in the async function.